data/meson.build

```diff
@@ -0,0 +1,32 @@
+# icons
+icon_dir = join_paths(datadir, 'icons', 'hicolor')
+icon_basename = '@0@'.format(project_name)
```

Comment:
project_name is already a string, I don't think you need to format it into a string?

Any reason you don't just use `project_name + '.svg'` down below?

Reply:
Because I wasn't thinking, I guess :(